### ISSUE TITLE
Time window for conversions

### ISSUE THREAD

**Issue by benab** [chacawaca/docker-post-recording#13]
I run this container on the same machine (possibly a mistake) as many other Docker images. Often the CPU is pegged due to a conversion running and it interferes with other containers that need the CPU.
Would the script inside the autovideoconverter folder be the one that I'd look to modify in order to only allow conversions to occur in a specific timeframe?

Thanks!

**Comment by chacawaca** [comment]
yes or you can modify your ffmpeg command to use nvidia or quicksync to save some cpu.

**Comment by benab** [comment]
Thanks for the tip. I don't think I can use nvidia or quicksync as I'm running Docker inside a virtual machine, so it doesn't have direct access to the hardware.

**Comment by Tonynks** [comment]
Can someone please help make a ffmeg command string for the custom.sh to be used with a Nas with a Celeron that has GPU.  I'm not sure what to change in the intel.sh example exactly. Thanks
